### PR TITLE
Portfolio: Add mobile menu

### PR DESCRIPTION
**Pull request by ernestmusong**
I have made the following changes to [my portfolio](https://ernestmusong.github.io/)

- Added the [HTML+CSS+JAVASCRIPT linter](https://github.com/ernestmusong/ernestmusong.github.io/blob/mobile-menu/.eslintrc.json)
- Added javascript [code]( https://github.com/ernestmusong/ernestmusong.github.io/blob/mobile-menu/index.js#:~:text=const%20sideBar%20%3D,Give%20feedback) to index.js file

**In index.html**

- Added a [semantic (aside) element for the mobile menu](https://github.com/ernestmusong/ernestmusong.github.io/commit/0ae1d7e03a2f01400b428abb6c352bc3bb786b53#:~:text=%3C!%2D%2D%20Mobile%20menu%20%2D%2D%3E,%3C/aside%3E)
**In index.css**

- Added [styling to the mobile menu](https://github.com/ernestmusong/ernestmusong.github.io/commit/8d9d8f4d7fd795647c27603b34acc7d2f999ea33#:~:text=%7D-,.show%2Dsidebar%20%7B,%7D,-/*%20Headline%20*/)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,6 +30,18 @@ <h3>My Logo</h3>
...
+            <button id="close-btn">
+                <i id="close" class="fas fa-times"></i>
+            </button>
+             <ul class="mobile-nav-menu">
+                <li class="nav-menu-item"><a class="nav-link" href="#">Portfolio</a</li>
+                <li class="nav-menu-item"><a class="nav-link" href="#about-me">About</a></li>
+                <li class="nav-menu-item"><a class="nav-link" href="#form-section">Contact</a></li> 
+                <li class="nav-menu-item"><a class="nav-link" href="#recent-works">Works</a></li> 
+            </ul>
+         </aside>
```
Comment (ShadyShawkat) [inline index.html:44]: - Please make sure to stick to the design as much as possible, which includes:
   - [ ] Adjusting the font size to better match the original design.
   - [ ] Changing the font color to better match the original design.
   - [ ] Removing the "works" nav link as that doesn't exist in the Figma template.
   - [ ] Removing the top border above the "portfolio" nav link.
   - [ ] Adjusting the positioning of the nav links and the spacing around them to better match the design.
   - [ ] Adjusting the color and size of the "close" button to match the original design.

| Your design | Figma design |
| --- | --- |
| image | image |

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ernestmusong) [reply index.html]: > * Please make sure to stick to the design as much as possible, which includes:
>   
>   * [ ]  Adjusting the font size to better match the original design.
>   * [ ]  Changing the font color to better match the original design.
>   * [ ]  Removing the "works" nav link as that doesn't exist in the Figma template.
>   * [ ]  Removing the top border above the "portfolio" nav link.
>   * [ ]  Adjusting the positioning of the nav links and the spacing around them to better match the design.
>   * [ ]  Adjusting the color and size of the "close" button to match the original design.
> 
> Your design	Figma design
> image	image

Thank you for your wonderful feedback @ShadyShawkat. I have now made all the changes outlined above.

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @ernestmusong, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**